proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,393 @@
+
+The LLVM Foundation accepts donations from sponsors and funds a variety of programs supporting the community and project, and controls project assets. Project assets include, but are not limited to, trademarks, web domains, and LLVM-affiliated accounts on platforms.
+
+The LLVM Foundation's core mission is to support the long-term health of the LLVM Project and Community. The LLVM Foundation does not drive the technical direction of the project, although the employees and directors of the LLVM Foundation have often been contributors and leaders in the project.
+
+As the owner of project assets, the Foundation gets the final say when allocating funding or decisions requiring legal oversight. The Foundation can approve or reject requests for funding on any basis it deems appropriate in accordance with applicable laws and policies.
+
+Members of the LLVM Foundation Board and employees of the LLVM Foundation are also members of the LLVM Community, as such they can also fill roles in any of the community structures defined in this proposal.
+
+
+### Modifications to the LLVM Decision Making Process
+
+The proposal process described in [LP0001 LLVM Decision Making](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md) should be adjusted in the following ways under this governance proposal.
+
+In step [3](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L60) of the process, in addition to identifying review managers, the proposal will identify the relevant _area team_ to oversee the proposal. If there is no relevant _area team_, or if the decision spans across the domains of multiple _area teams_ the _project council_ will oversee the proposal.
+
+In steps [4](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L61) and [8](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L65), the roles designated to Chris Lattner are replaced by the overseeing _area team_ or the _project council_.
+
+## Voting Process
+
+Public notice of any vote will be published on the LLVM discussion forums under the "Elections" topic at least 14 days before voting opens with a schedule for the election. Each election will be open for 14 days. Reminders will be posted on the "Elections" topic at:
+
+* 7 days before voting opens.
+* The day voting opens.
+* 7 days after voting opens.
+* 48 hours before voting closes.
+* 24 hours before voting closes.
+
+Voter eligibility will be determined 7 days before voting opens by the rules for defining _voting contributors_. Voting will be executed using a private poll hosted by [Concordcet Internet Voting Service]([https://civs1.civs.us/](https://civs1.civs.us/)). Concordcet uses a ranked choice polling process.
```

Comment:
FYI, CIVS has been used in the past by a member of the open-source community to send abusive polls to other open-source members (see https://github.com/rms-support-letter/rms-support-letter.github.io/issues/6450 for example, though much of what I know is from private discussions). When CIVS was contacted about the abuse, the response was to filter the emails yourself, and so there is a good chance some members of our community are doing so and may not get notifications from this system. https://elekto.dev seems to be an example of a more modern site that's not so open for abuse, but there may be others, and I have no idea as to the quality of that one.

Reply:
That is great to know. I have no experience with CIVS, I just saw that other open source communities used it. Looking at electo.dev, it looks neat, but relies on OAuth rather than email, which I'm fine with, but it has impact on how we handle voter registration and communications.

I'm going to add a note to the document that CIVS is a placeholder and we need to do a more through evaluation of our options.